src/transformers/models/blenderbot/modeling_blenderbot.py

```diff
@@ -1576,9 +1576,12 @@ def forward(
             cross_attentions=outputs.cross_attentions,
         )
 
+    # Ignore copy
     def prepare_inputs_for_generation(
         self, input_ids, past_key_values=None, attention_mask=None, use_cache=None, **kwargs
     ):
+        # Overwritten -- model logic breaks when `inputs_embeds` are passed from this function
+
```

Comment:
Just curious: does that mean blenderbot cannot generate from inputs embeds and it cannot be fixed? I see many models touched here didn't pass further inputs embeds, so that mean after this PR all of them will support generation from embeddings. So interesting to see why this model failed

Reply:
>  I see many models touched here didn't pass further inputs embeds, so that mean after this PR all of them will support generation from embeddings.

Precisely! Many models will get this feature for free as part of these deletions 💛 

> Just curious: does that mean blenderbot cannot generate from inputs embeds and it cannot be fixed?

No clue, I didn't dive deeper :) Failed in `inputs_embeds` tests -> pasted this comment. I don't think these combos of model/feature are worth the dive, so I left this low-information (but better than nothing) note

Reply:
Actually the test was just flaky! I've added flakiness protection to the failing test and deleted a few more cases :)